test/transforms/test_image_masking_transform.py

```diff
@@ -4,21 +4,19 @@
 # This source code is licensed under the BSD-style license found in the
 # LICENSE file in the root directory of this source tree.
 
-import unittest
-
+import pytest
 import torch
-from test.test_utils import assert_expected, get_asset_path
-from torchmultimodal.transforms.image_masking_transform import (
-    MaskedImageModelingTransform,
-)
+from test.test_utils import assert_expected, set_rng_seed
+from torchmultimodal.transforms.flava_transform import MaskedImageModelingTransform
 from torchvision import transforms
 
 
-class TestImageMaskingTransforms(unittest.TestCase):
-    def setUp(self):
-        self.test_image = get_asset_path("grace_hopper_517x606.jpg")
+class TestImageMaskingTransforms:
+    @pytest.fixture
```

Comment:
nit: You can set `autouse=True` for seed setting such that you won't need to request it for any test/fixture that depends on it.